repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/keystonev2/timeout/httpConnTimeout60sec/http-connection-pool.cfg.xml

```diff
@@ -1,27 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <http-connection-pools xmlns="http://docs.openrepose.org/repose/http-connection-pool/v1.0">
     <pool id="default"
-          default="true"
+          default="false"
```

Comment:
these too

Reply:
Same thing as above. I think this is the one that GitHub confused the other for.